recipes/pro-mdnsd/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+import os
+import sys
+from conans import ConanFile, CMake, tools
+from conans.tools import download, unzip
+from conans.model.version import Version
+
+
+class mdnsdConan(ConanFile):
+    name = "pro-mdnsd"
+    license = "BSD 3-Clause"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Please use SPDX license identifier https://spdx.org/licenses/BSD-3-Clause.html